code/billy/lab_04.py

```diff
@@ -6,14 +6,16 @@
 funcion: this program will generate 6 random numbers as the "winning numbers". The net winnings will be printed out at the end.
 
 '''
-
 import random
 
+# the program rarely returns matching numbers, for some reason.
```

Comment:
[Lab 4: Pick6](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/04%20Pick%206.md)

* Needs Work:

* Expected ROI is around -.88

* Sample output:
```
PS C:\Users\Bruce\Programming\class_swordfish\code\billy> python .\lab_04.py

> You have 0 matches

> Your total earnings: -200000

> your ROI is:  -1.0

PS C:\Users\Bruce\Programming\class_swordfish\code\billy>
```